Scripts/Python/grsnNexusBookMachine.py

```diff
@@ -174,11 +200,11 @@ def OnNotify(self,state,id,events):
 
         if (id == booksouthInPos.id and master):
             PtDebugPrint("South book aligned")
-            booksouthOutResponder.run(self.key)
+            self.SendNote('SouthBook')
 
         if (id == booknorthInPos.id and master):
             PtDebugPrint("North book aligned")
-            booknorthOutResponder.run(self.key)
+            self.SendNote('NorthBook')
```

Comment:
Needed so that the nexus book pillar functions correctly in a multiplayer environment. It didnt center correctly when the age owner was syncing the animations to other players. What it does now is have the age owner trigger a notify to all players that the book sequence is happening and the responder plays locally for everyone.

Reply:
There should be no difference in those two situations...

Reply:
And yet in testing on staging there was... Book coming out slightly offcenter to very offcentered over time